src/Extension/CoreExtension.php

```diff
@@ -364,7 +364,6 @@ function twig_random(Environment $env, $values = null, $max = null)
             }
         } else {
             $min = $values;
-            $max = $max;
```

Comment:
The left-hand and right-hand sides of the assignment are equal.

Added by [#2698](https://github.com/twigphp/Twig/pull/2698/files#diff-2fa56c635b23777069e2e6336d62e65fa50dee4e6dc76ca60ab9383c39b50ef9R356), perhaps to be explicit about the value of `$max`.